app/src/main/java/com/example/abhishek/bookshareapp/utils/CommonUtilities.java

```diff
@@ -7,5 +7,5 @@
 
     //public static final String google_api_url="https://www.googleapis.com/";
     public static final String goodreads_api_url="https://www.goodreads.com/";
-    public static final  String API_KEY= "ATQ5fPyd24lOEhsElf860w";
+    public static final  String API_KEY= "Q4TqWuuk0j2E2LKvowRRpg";
```

Comment:
You should ideally not put your keys public  :sweat_smile: Don't let them out on Github. @abhisheksachdeva Please help him out